pkg/sql/types/types.go

```diff
@@ -1102,7 +1102,7 @@ func (t *T) StableTypeID() uint32 {
 
 // UserDefined returns whether or not t is a user defined type.
 func (t *T) UserDefined() bool {
-	return t.StableTypeID() != 0
```

Comment:
hmm, does this no longer work?

Reply:
The ambiguous case for enums (and probably other UDT's) AnyEnum doesn't have a stable type ID. I don't want to always special case it like that, so I think just checking families here makes more sense.